### DISCUSSION TITLE
[FileInput] Remove :hover and :active FileItemIconBackgroundColor on error

### DISCUSSION THREAD

**Discussion by aweell** [dxc-technology/halstack-react#507] [Ideas] [answered]
Instead of having particular tokens for the fileItemIcon background on error:

* `errorHoverDeleteFileItemBackgroundColor`
* `errorActiveDeleteFileItemBackgroundColor`

Use the transparent scale in the background color to cover both cases:

Component token | Element | Core token | Value
-- | -- | -- | --
`hoverDeleteFileItemBackgroundColor`  | File item icon:hover | `color-grey-a-100` | `#0000000d`
`activeDeleteFileItemBackgroundColor` | File item icon:active | `color-grey-a-300` | `#00000033`

Removing the error tokens listed at the top and keeping only two.

![Screen Shot 2022-01-18 at 12 59 12 PM](https://user-images.githubusercontent.com/44420072/149933548-9256150c-0a3f-41d6-b6d5-38598f21a7b4.png)

**Comment by GomezIvann** [comment, marked as answer]
I'm gonna open an issue for this!

**Reply by GomezIvann** [reply]
This discussion led to an issue and is already done, marked this as answered.
